src/plotting.jl

```diff
@@ -7,19 +7,20 @@
     principalcoord(pc, 1), principalcoord(pc,2)
 end
 
-@recipe function f(abun::AbundanceTable; topabund::Int=10, sorton::Symbol=:top)
```

Comment:
Recipes currently fail on type assertions for keyword args, though a fix is in the works.